src/frontend/src/views/GroupMembersPage/components/MembersTab/components/InviteModal/index.tsx

```diff
@@ -48,12 +51,35 @@ const InviteModal = ({ groupName, onClose, open }: Props): JSX.Element => {
     onClose();
   };
 
-  const getAddressArrayFromInputValue = () => {
-    return uniq(inputValue.trim().split(INPUT_DELIMITERS));
+  /**
+   * Gets user input and chunks into (what should be) email addresses.
+   *
+   * Original user input text is broken into chunks by delimiters. Repeated
+   * delimiters between chunks are taken as a single delimiter. Leading and
+   * trailing delimiters are dropped. If a given chunk is found multiple times
+   * in the input, only one copy of the chunk is returned in output array.
+   *
+   * Example:
+   *     user1@example.com,   user2@example.com, ,, ,,  , user3@example.com,,
+   *   user2@example.com, user4@example.com,,
+   * Would return as four emails in total, as you'd expect to read them.
+   *
+   * Arguable if it's best UX -- should double commas bring up an error? etc --
+   * but it's easiest to code and reasonably sane to just ignore that sort
+   * of stuff. Approach was approved by Design.
+   */
+  const getAddressArrayFromInputValue = (): string[] => {
+    // First pass may have multiple spaces between chunks or pre/suffix spaces
+    const delimitBySpaces = inputValue.replace(INPUT_DELIMITERS, " ");
+    const delimitBySingleSpaceTrimmed = delimitBySpaces
```

Comment:
why re-assign to `delimitBySpaces` to `delimitBySingleSpaceTrimmed `?

Reply:
Sorry, I don't follow? It appears to be a weird auto-styling thing, but the methods that get called show up on the next line, so while it looks like a simple aliasing at first glance, it's actually running all those methods on original `delimitBySpaces` before setting result as `delimitBySingleSpaceTrimmed`. Does that clear it up, or did you mean something else, like why not just do it all in a single giant run of method calls? (which could be done, I just figured having an intermediate var made the process a bit easier to follow)